<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="buttonSongDatabase">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/icons/icons/song1.png</normaloff>:/icons/icons/song1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDateDatabase">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/icons/icons/calendaricons.png</normaloff>:/icons/icons/calendaricons.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>196</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="SongDatabaseWidget" name="songDatabaseWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QTreeView" name="treeView">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>120</y>
           <width>256</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Song"/>
   <addaction name="actionDelete_Song"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Attachment"/>
   <addaction name="actionDelete_Attachment"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPush"/>
    <addaction name="actionPull"/>
    <addaction name="actionClone"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAttachments">
    <property name="title">
     <string>Attachments</string>
    </property>
    <addaction name="actionNew_Attachment"/>
    <addaction name="actionDelete_Attachment"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QMenu" name="menuUnzip_directory">
     <property name="title">
      <string>Unzip directory:</string>
     </property>
     <widget class="QMenu" name="menu_path">
      <property name="title">
       <string>&lt; path &gt;</string>
      </property>
      <addaction name="actionOpen_Terminal_here"/>
      <addaction name="actionCopyToClipboard"/>
     </widget>
     <addaction name="menu_path"/>
    </widget>
    <addaction name="menuUnzip_directory"/>
   </widget>
   <widget class="QMenu" name="menuFile_Index">
    <property name="title">
     <string>File Index</string>
    </property>
    <addaction name="actionUpdate_Index"/>
    <addaction name="actionAdd_Folder"/>
    <addaction name="actionClear_Index"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAttachments"/>
   <addaction name="menuTools"/>
   <addaction name="menuFile_Index"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionNew_Song">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/oldIcons/oldIcons/1411698488_document-add-128.png</normaloff>:/oldIcons/oldIcons/1411698488_document-add-128.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Song</string>
   </property>
   <property name="toolTip">
    <string>Creates a new song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/floppy16.png</normaloff>:/icons/icons/floppy16.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/floppy16.png</normaloff>:/icons/icons/floppy16.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/open127.png</normaloff>:/icons/icons/open127.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUpdate_Index">
   <property name="text">
    <string>Update Index</string>
   </property>
  </action>
  <action name="actionAdd_Folder">
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionClear_Index">
   <property name="text">
    <string>Manage Index</string>
   </property>
   <property name="toolTip">
    <string>Clear Index</string>
   </property>
  </action>
  <action name="actionNew_Attachment">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/oldIcons/oldIcons/1411698394_attachment_add-128.png</normaloff>:/oldIcons/oldIcons/1411698394_attachment_add-128.png</iconset>
   </property>
   <property name="text">
    <string>New Attachment </string>
   </property>
  </action>
  <action name="actionDelete_Attachment">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/oldIcons/oldIcons/1411698394_attachment_remove-128.png</normaloff>:/oldIcons/oldIcons/1411698394_attachment_remove-128.png</iconset>
   </property>
   <property name="text">
    <string>Delete Attachment</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/left64.png</normaloff>:/icons/icons/left64.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/right53.png</normaloff>:/icons/icons/right53.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete_Song">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/oldIcons/oldIcons/1411698488_document-delete-128.png</normaloff>:/oldIcons/oldIcons/1411698488_document-delete-128.png</iconset>
   </property>
   <property name="text">
    <string>Delete Song</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionPush">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/up51.png</normaloff>:/icons/icons/up51.png</iconset>
   </property>
   <property name="text">
    <string>Push ...</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/down27.png</normaloff>:/icons/icons/down27.png</iconset>
   </property>
   <property name="text">
    <string>Pull</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/war4.png</normaloff>:/icons/icons/war4.png</iconset>
   </property>
   <property name="text">
    <string>Clone ...</string>
   </property>
  </action>
  <action name="actionOpen_Terminal_here">
   <property name="text">
    <string>Open terminal</string>
   </property>
  </action>
  <action name="actionCopyToClipboard">
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SongDatabaseWidget</class>
   <extends>QWidget</extends>
   <header>songdatabasewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
